docs/pages/includes/sso/loginerrortroubleshooting.mdx

```diff
@@ -58,3 +58,12 @@ spec:
 version: v5
 ```
 
+### Single sign-on fails with OIDC
+
+The error typically indicates a discrepancy between the algorithm used to sign the JWT token and the algorithm(s) supported by the JSON Web Key Set (JWKS). Specifically, the token might be signed with one algorithm, e.g., HS256, while the JWKS only lists keys for a different algorithm. e.g., RS256. This issue predominantly arises when using identity providers that offer extremely low-level functionality.
```

Comment:
I think we should still mention the actual error that the user will see in this case (Failed to verify JWT: oidc: unable to verify JWT signature: no matching keys), my suggestion was just that we don't make it the title of the section.